core/src/main/java/jenkins/security/apitoken/TokenUuidAndPlainValue.java

```diff
@@ -24,13 +24,13 @@
 package jenkins.security.apitoken;
 
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.accmod.restrictions.Beta;
 
 /**
  * Contains information about the token and the secret value.
  * It should not be stored as is, but just displayed once to the user and then forget about it.
  */
```

Comment:
```suggestion
 * @since TODO
 */
```

since you are effectively exposing this class (and thus, the method that returns it) for the first time.